manifests/post-installation/ovn-template.yaml

```
@@ -0,0 +1,21 @@
+      repoURL: oci://quay.io/szigmon
+      chart: ovn
+      version: "v25.4.0-custom"
+    values:
+      commonManifests:
+        enabled: true
+      dpuManifests:
+        enabled: true
+        cniBinDir: /var/lib/cni/bin/
+        cniConfDir: /run/multus/cni/net.d
+      leaseNamespace: "ovn-kubernetes"
+      gatewayOpts: "--gateway-interface=br-ovn --gateway-uplink-port=puplinkbrovn"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)